packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/components/ChoiceButton.js

```diff
@@ -5,113 +5,157 @@ import {
+          light: 'neutral-6'
+        }
+  const border = selected
+    ? { color: 'neutral-6', size: '1px' }
+    : undefined
+  const textColor = selected 
+    ? 'neutral-6' 
+    : {
+        dark: 'neutral-6',
+        light: 'neutral-7'
+      }
+  const thumbnailHeight = 150
   const thumbnailWidth = Math.round(thumbnailHeight * THUMBNAIL_ASPECT_RATIO)
   const thumbnailSrc = `https://thumbnails.zooniverse.org/${thumbnailWidth}x${thumbnailHeight}/${src.slice(8)}`
   
   return (
-    <Button
-      ref={choiceButton}
-      aria-checked={ariaChecked}
-      disabled={disabled}
+    <StyledBox
+      ref={choiceMenuItem}  
+      // TODO: add the following to translations
+      a11yTitle={`${choiceLabel}` + (selected ? '; identified' : '')}
+      aria-haspopup='true'
+      role='menuitem'
+      align='center'
+      background={background}
+      border={border}
+      direction='row'
       fill
-      label={
-        <StyledBox
-          align='center'
-          columnsCount={columnsCount}
-          fill
-          forwardedAs='span'
+      onKeyDown={disabled ? DEFAULT_HANDLER : handleKeyDown}
+      pad={{
+        right: '10px',
+        vertical: '5px'
+      }}
+      selected={selected}
+      tabIndex={tabIndex}
+    >
+      {selected ? (
+        <DeleteButton
+          choiceLabel={choiceLabel}
+          deleteFn={handleDelete}
+          disabled={disabled}
+          tabIndex={selected && tabIndex === 0 ? 0 : -1}
         >
           {thumbnailSize !== 'none' && src &&
-            <Image
+            <StyledImage
               alt=''
-              height='fill'
+              height='50'
               src={thumbnailSrc}
-              width={thumbnailWidth}
+              width={thumbnailSize === 'small' ? '50' : '60'}
             />}
-          <Text>
-            {choiceLabel}
-          </Text>
-        </StyledBox>
-      }
-      onClick={handleClick}
-      onKeyDown={handleKeyDown}
-      role={role}
-      selected={selected}
-      size='small'
-      tabIndex={tabIndex}
-    />
+        </DeleteButton>
+      ) : null}
+      <StyledButton
+        // TODO: add the following to translations
+        a11yTitle={`Open submenu for ${choiceLabel}`}
+        disabled={disabled}
+        label={
+          <Box
+            align='center'
+            direction='row'
+            forwardedAs='span'
+          >
+            {!selected && thumbnailSize !== 'none' && src &&
+              <StyledImage
+                alt=''
+                height='50'
+                src={thumbnailSrc}
+                width={thumbnailSize === 'small' ? '50' : '60'}
+              />}
+            <Text
+              color={textColor}
+              margin={{ left: '10px', vertical: '5px' }}
+              size={thumbnailSize === 'small' ? '.875rem' : '1rem'}
+              weight={selected ? 'bold' : 'normal'}
+              wordBreak='break-word'
+            >
+              {choiceLabel}
+            </Text>
```

Comment:
Regarding ChoiceButton labels, I think the styling should be as follows, but @seanmiller26 could you please confirm or let me know if I'm forgetting any edge cases:
- if a ChoiceButton label exceeds the button width, it should wrap to a second line
- if a ChoiceButton label exceeds the button width and a second line, whatever exceeds the second line should be hidden (overflow hidden)

Reply:
If a label is more than 2 lines, hide the overflow. We should try it out and see what it looks like. Possibly with ellipse after a certain character count.